addon/services/page-title-list.js

```diff
@@ -205,12 +222,26 @@ export default Service.extend({
     return title.join('');
   },
 
+  _updateTitle() {
+    const toBeTitle = this.toString();
+
+    if (isFastBoot) {
+      // in fastboot context "document" is instance of ember-fastboot/simple-dom document
+      let titleEl = this.document.createElement('title');
+      let titleContents = this.document.createTextNode(toBeTitle);
+      titleEl.appendChild(titleContents);
+      this.document.head.appendChild(titleEl);
```

Comment:
Could this end up in a scenario where there is more than one title? Esp if there was one already?

Reply:
`ember-cli-fastboot` blueprint will/should be updated - so that upon install default index.html <title> is replaced with title in application.hbs. Bascially for Fastboot requirements about title element removal are the same that are already in README of this repository.

Even if there are more than one title element, in Fastboot context SimpleDOM document has now access to mutate original HTML.

Reply:
We could add linter rule for "no-fastboot-index-title" eventually as well. 🤔

Reply:
> ember-cli-fastboot blueprint will/should be updated - so that upon install default index.html <title> is replaced as title in application.hbs. Basically for Fastboot requirements about title element removal are the same that are already in README of this repository.

I think both blueprints (ember-cli-fastboot and ember-page-title) could do the check to see if both `ember-cli-fastboot` and `ember-page-title` are present and if so remove the `<title></title>` from `app/index.html`. 

Mind making a meta issue for that here, and cross linking in an issue in `ember-cli-fastboot`?

Reply:
Issues created: 

* https://github.com/adopted-ember-addons/ember-page-title/issues/174
* https://github.com/ember-fastboot/ember-cli-fastboot/issues/792